optuna/integration/tfkeras.py

```diff
@@ -40,8 +37,7 @@ def __init__(self, trial, monitor):
         self._trial = trial
         self._monitor = monitor
 
-    def on_epoch_end(self, epoch, logs=None):
-        # type: (int, Dict[str, Any]) -> None
+    def on_epoch_end(self, epoch: int, logs: Dict[str, Any] = None) -> None:
```

Comment:
Stretches the scope of this PR so it's optional, but we could also correct the type hint here (`Optional` since it can take the value `None`). C.f. https://github.com/optuna/optuna/pull/1703
```suggestion
    def on_epoch_end(self, epoch: int, logs: Optional[Dict[str, Any]] = None) -> None:
```
Note that `typing.Optional` must be imported in addition to the suggestion.